test/sharness/t0119-prometheus-data/prometheus_metrics

```diff
@@ -94,7 +96,6 @@ go_memstats_heap_objects
 go_memstats_heap_released_bytes
 go_memstats_heap_sys_bytes
 go_memstats_last_gc_time_seconds
-go_memstats_lookups_total
```

Comment:
:information_source: Rationale for removal in https://github.com/prometheus/client_golang/releases/tag/v1.20.0 ("was broken anyway")